
uint8 id

uint16 close_position
uint16 close_speed
uint16 close_hold_torque
uint16 close_max_torque
---
uint8 state
